### PR TITLE
JavaScript3-Homework-Week2

### PR DESCRIPTION
**Pull request by qamarfj**

### CODE REVIEW

File: javascript/javascript3/week2/movies-exercise/movies.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+fetch(
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+)
+  .then((response) => response.json())
+  .then((movies) => {
+    //bad movies
+    const badMovies = movies.filter((movie) => movie.rating < 3);
+    //bad movies Since 2000
+    const badoMviesSince2000 = badMovies.filter((movie) => movie.year >= 2000);
```
Comment (zuhairtaha) [inline javascript/javascript3/week2/movies-exercise/movies.js]: Just a variable spell mistake

File: javascript/javascript3/week2/promise-visual/main-then.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        console.log("RED Element has been moved");
+        moveElement(blueBox, targetToMove.blue).then(() => {
+          console.log("Blue Element has been moved");
+          moveElement(greenBox, targetToMove.green).then(() => {
+            console.log("Greeen Element has been moved");
+          });
+        });
+      })
+      .catch((error) => console.log(error));
+  }
```
Comment (zuhairtaha) [inline javascript/javascript3/week2/promise-visual/main-then.js:33]: You can also return a promise for each box and continue chaining. Similar to this
```js
function translateOneByOne() {
  moveElement(red, target.red)
    .then(() => {
      console.log('red moved');
      return moveElement(blue, target.blue);
    })
    .then(() => {
      console.log('blue moved');
      return moveElement(green, target.green);
    })
    .then(() => {
      console.log('green moved');
      return;
    });
// ...
}
```

**Review by zuhairtaha — APPROVED** [review]
Perfect Qamar. Well done. 
I added simple notes. But in general you can merge it. Everything seems fine 👍

File: javascript/javascript3/week2/promise-visual/main-then.js (hunk 2 above)
Comment (qamarfj) [reply javascript/javascript3/week2/promise-visual/main-then.js:33]: thank you for your precious time for the review and valuable comments.

Comment (zuhairtaha) [reply javascript/javascript3/week2/promise-visual/main-then.js:33]: My pleasure ☺

**Pull request merged**